### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by ramiaquista**
## What I have done 🔨  :
- Added structure to the application folders/files.
- Added 2 components/ BookPage() and CategoriesPage().
- Add a list and a form to BookPage React Component.
- Added a list and a form to BookPage() React Component.
- Added styling to CategoriesPage() inside CategoriePageStyle.css file.
- Added React Router inside App() Component to switch between pages.
- Added Bootstrap import on BookPage.js and CategoriesPage.js
- Changed the title of the website.
- Added Book(), CreateNewBook() components

### CODE REVIEW

File: src/components/BooksPage.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+const BooksPage = () => (
+  <main>
+    <section className="books-page">
+      <h1>Books List</h1>
+      <ListGroup>
+        <ListGroup.Item>
+          Cras justo odio
+          <Button variant="secondary">Remove</Button>
+        </ListGroup.Item>
```
Comment (helman101) [inline src/components/BooksPage.js]: - [x] Please create a separate component for the books, for example a `<Book />` component 😄

File: src/components/BooksPage.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            <Form.Label>Add new book</Form.Label>
+            <Form.Control type="email" placeholder="Book title" />
+          </Form.Group>
+          <Form.Select aria-label="Default select example">
+            <option>Category</option>
+            <option value="1">Adventure</option>
+            <option value="2">Classics</option>
+            <option value="3">Horror</option>
+          </Form.Select>
+        </Form>
```
Comment (helman101) [inline src/components/BooksPage.js]: - [x] Your form should be in a separate component, kindly move this to another file, for example, a `CreateNewBook.js` file 😄

File: src/components/BooksPage.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+          <Button variant="secondary">Remove</Button>
+        </ListGroup.Item>
+        <ListGroup.Item>
+          Porta ac consectetur ac
+          <Button variant="secondary">Remove</Button>
+        </ListGroup.Item>
+        <ListGroup.Item>
+          Vestibulum at eros
+          <Button variant="secondary">Remove</Button>
+        </ListGroup.Item>
```
Comment (helman101) [inline src/components/BooksPage.js]: - [x] The data of your books should be passed as a `prop`, like this:
    - `<Book title={books[0]} />`, I would recommend you to have a list of books on this component, in order to pass data to each `<Book/>` component 😄

**Review by helman101 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🔴 

Hi @ramiaquista  👋, 

Nice job so far with this project 😄, but there are still some changes to do.

## Required Changes ♻️

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BooksPage.js (hunk 3 above)
Comment (ramiaquista) [reply src/components/BooksPage.js]: Thanks, @helman101  for the review! 😄

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉 

## Hi @ramiaquista, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**